ports/kfr/portfile.cmake

```
@@ -1,23 +1,25 @@
 vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "Linux")
 
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO kfrlib/kfr
-    REF 1f9706197abfcd4b4ec19ded3ce37b70ebd9a223
-    SHA512 901c6984a46a7abcc28adf9397759156a9e8d173e028c236ab423568ed20b3a3efe207be9660c961539c73a2767afaedcd76133304f542d3299353942cf13f5e
+    REF 9fc73247f43b303617329294ae264613df4dce71 # 4.2.1
+    SHA512 c7dd4b1a0be436460973fb8a48bc6f2264a0f7d8d034ce88ccfd8328135f1492eab155023103a1461c2058eb6c79a6019b62d023dc5bc390ab4d2b43eac9c2d4
     HEAD_REF master
 )
 
 vcpkg_check_features(
     OUT_FEATURE_OPTIONS FEATURE_OPTIONS
-    capi ENABLE_CAPI_BUILD
-    dft ENABLE_DFT
-    dft-np ENABLE_DFT_NP
+	FEATURES
+        capi ENABLE_CAPI_BUILD
+        dft ENABLE_DFT
+        dft-np ENABLE_DFT_NP

```

```suggestion
    FEATURES
        capi ENABLE_CAPI_BUILD
        dft ENABLE_DFT
        dft-np ENABLE_DFT_NP
```
The `FEATURES ` should be align left with `OUT_FEATURE_OPTIONS `.

There should be four spaces at the left of these feature options.

---

Hi! @NancyLi1013 Thanks. Fixed.